uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Service/DrupalUserService.php

```diff
@@ -89,6 +89,19 @@ public function addRoles($email, $roles) {
     user_save($user, ['roles' => $roles]);
   }
 
+  /**
+   * Defaults user's role to supplied role(s)
+   *
+   * @param string $email
+   * @param array $roles
+   */
+  public function setRoles($email, $roles) {
```

Comment:
Please add a test for this method.

Reply:
After some discussion with @ajesamson , its a bit difficult to mock tests for this so approving this without a test.